### PR TITLE
Reformat env vars table for OTLP exporter.

### PR DESCRIPTION
**Pull request by anuraaga**
## Changes

No change in content just moves the items into a list form to make it easier to add new entries as found in #2330

### CODE REVIEW

**Review by pirgeo — APPROVED** [review]
I think putting the bullet point headings (**Endpoint (OTLP/HTTP**, **Insecure**, etc.)  in bold might be a good idea, just to have some visual cue when looking for something on this page. This is what I would expect people to use this page for.

**Review by yurishkuro — APPROVED** [review]
+1 to bold the entry titles (as was suggested in my original comment)

**Review by tigrannajaryan — APPROVED** [review]

File: specification/protocol/exporter.md [hunk 1]
```diff
@@ -8,16 +8,37 @@ This document specifies the configuration options available to the OpenTelemetry
 
 The following configuration options MUST be available to configure the OTLP exporter. Each configuration option MUST be overridable by a signal specific option.
```
[the remark this answers is not in the document]
Comment (anuraaga) [reply specification/protocol/exporter.md:9]: I think it's referring to how each env var has a form FOO_BAR, FOO_TRACES_BAR, FOO_METRICS_BAR

**Comment by tigrannajaryan** [comment]
@anuraaga this should be good to merge once rebased and all comments resolved.

File: specification/protocol/exporter.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (yurishkuro) [reply specification/protocol/exporter.md:9]: resolving since this PR is not changing the content, only formatting

**Pull request merged**